modules/cli/src/main/scala/scala/cli/commands/shared/SharedCompilationServerOptions.scala

```diff
@@ -276,7 +276,7 @@ final case class SharedCompilationServerOptions(
       javaOpts =
         (if (bloopDefaultJavaOpts) baseConfig.javaOpts
          else Nil) ++ bloopJavaOpt ++ bloopDefaultJvmOptions(logger).getOrElse(Nil),
-      minimumBloopJvm = javaV.getOrElse(8),
```

Comment:
weren't we set on allowing to run Bloop with older Java versions? I have a vague recollection it should still run on 11 🤔 
I could be wrong

Reply:
Right now, we very restricitvely keep the JVM for bloop at least version 17, so I thought using 17 as minimum here is reasonable.
If the minimum is not set there's a possibility that an existing bloop server won't be restarted if it's jvm is older than requested. Actually that was part of the problem of the issue.

Reply:
Alright then, let's bump to 17 👍